script/releases/v0.1.0-rewardsv2/3-multisig.s.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {MultisigCall, MultisigCallUtils, MultisigBuilder} from "zeus-templates/templates/MultisigBuilder.sol";
+import {SafeTx, SafeTxUtils} from "zeus-templates/utils/SafeTxUtils.sol";
+import {ITimelock} from "zeus-templates/interfaces/ITimelock.sol";
+import {Queue} from "./2-multisig.s.sol";
+import {EigenLabsUpgrade} from "../EigenLabsUpgrade.s.sol";
+
+contract Execute is Queue {
+    using MultisigCallUtils for MultisigCall[];
+    using SafeTxUtils for SafeTx;
+    using EigenLabsUpgrade for *;
+
+    MultisigCall[] private _opsCalls;
+
+    /**
+     * @dev Overrides the previous _execute function to execute the queued transactions.
+     */
+    function _execute() internal override returns (MultisigCall[] memory) {
+        MultisigCall[] memory _executorCalls = _queue();
+
+        bytes memory executorCalldata = _executorCalls.makeExecutorCalldata(this._multiSendCallOnly(), this._timelock());
+        _opsCalls.append({
+            to: this._timelock(),
+            value: 0,
+            data: abi.encodeWithSelector(ITimelock.executeTransaction.selector, executorCalldata)
+        });
+
+        return _opsCalls;
+    }
+
+    function testDeployAgain() public {
+        _execute();
+    }
```

Comment:
We should test that the new initialized variables are correct here (i.e the ones set in `initialize`). 

We should also test that the new storage variables added in Rewards v2 upgrade are now available.